src/compiler/types.rs

```diff
@@ -1071,11 +1112,13 @@ pub trait TypeCheckerHost: ModuleSpecifierResolutionHost {
     fn get_source_files(&self) -> Vec<Rc<Node>>;
 }
 
+#[derive(Debug)]
 #[allow(non_snake_case)]
 pub struct TypeChecker {
     pub _types_needing_strong_references: RefCell<Vec<Rc<Type>>>,
-    pub Symbol: fn(SymbolFlags, __String) -> Symbol,
+    pub Symbol: fn(SymbolFlags, __String) -> BaseSymbol,
     pub Type: fn(TypeFlags) -> BaseType,
+    pub(crate) empty_symbols: Rc<RefCell<SymbolTable>>,
```

Comment:
This isn't different than the others, I think `pub(crate)` would probably make more sense for all of these `TypeChecker` members (they just need to be accessible from the type-checking code)